src/aead/chacha20_poly1305_openssh.rs

```diff
@@ -26,7 +26,8 @@
 //! `padding_length||payload||random padding`.
 //!
 //! [chacha20-poly1305@openssh.com]:
-//!    http://cvsweb.openbsd.org/cgi-bin/cvsweb/src/usr.bin/ssh/PROTOCOL.chacha20poly1305?annotate=HEAD
+//!    http://cvsweb.openbsd.org/cgi-bin/cvsweb/src/usr.bin/ssh/\
```

Comment:
This url-line-breaking is pretty unfortunate but I didn't see any other way to appease rustfmt.